livy/src/test/java/org/apache/zeppelin/livy/LivyInterpreterIT.java

```diff
@@ -379,6 +379,15 @@ public void testPySparkInterpreter() throws InterpreterException {
     } catch (Exception e) {
       e.printStackTrace();
     }
+        //test special characters
+    try {
```

Comment:
Please remove the try catch as the exception here is ignored. Could you also help to remove the try catch in the above unicode test ?

Reply:
Hi @zjffdu  ,

I have implemented the review comment , and commited along with this PR, Please feel free to review.